framework/templates/ecl-comments-with-form/ecl-comments-with-form.twig

```diff
@@ -0,0 +1,90 @@
+{#
+  Accepts:
+    - "image" (object): image to display on the left (optional)
+      - "src" (string): image's URL
+      - "alt" (string): alt text
+    - "author" (string): author name
+    - "date" (string): formatted date
+    - "title" (object): comment's title
+      - "href" (string): link to point to
+      - "text" (string): text to display
+    - "body" (string): comment's text
+    - "operations" (object): object of operations (reply, edit, delete)
+      - "delete|edit|reply" (object):
+        - href (string)
+        - title (string)
+        - visible (boolean)
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-row ecl-comments-with-form' %}
```

Comment:
- `ecl-comments-with-form` doesn't exist
- `ecl-row` should not be used here (no cols)

Reply:
Please remove `_css_class` and `_extra_attributes` altogether. This is an example template, it doesn't need to be extensible ;)